src/renderer/components/download-settings/download-settings.vue

```diff
@@ -34,6 +34,7 @@
             :show-action-button="false"
             :show-label="false"
             :disabled="true"
+            :maxlength="100"
```

Comment:
As this displays the users chosen downloads folder, the path could much longer than 100 characters.
```suggestion
```